### PR TITLE
Setup to add this project at Packagist

### PR DESCRIPTION
**Pull request by kamihouse**
Add settings to deliver this project in Packagist.

### CODE REVIEW

**Comment by kamihouse** [comment]
Link: [composer require igorescobar/jquery-mask-plugin](https://packagist.org/packages/igorescobar/jquery-mask-plugin)
Caso seja do seu interesse aceitar meu pull request, após sair a nova versão (contendo o `composer.json`) altero para o link do repositório original e passo as instruções de configuração do webhook, para manter o pacote sempre atualizado no [Packagist](https://packagist.org).

Ps: geralmente o pessoal opta por utilizar o `npm` para gerenciar dependências JS, pela importância desse plugin eu prefiro que ele também esteja disponível no `composer`.
Um abraço.

File: composer.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{
+  "name": "igorescobar/jquery-mask-plugin",
```
Comment (igorescobar) [inline composer.json]: Change it to jquery-mask-plugin. And I will setup the web hook. Already got an account. @igorescobar.

Comment (igorescobar) [inline composer.json]: Web hook is already configured but its disabled. Will activate when merged.

File: composer.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  "homepage": "https://github.com/igorescobar/jQuery-Mask-Plugin",
+  "license": "MIT",
+  "authors": [
+    {
+      "name": "Igor Escobar",
+      "email": "blog@igorescobar.com",
+      "homepage": "https://about.me/igorescobar",
+      "role": "Developer"
+    },
+    {
```
Comment (igorescobar) [inline composer.json]: I don't think its fair to add only you as a contributor. Maybe add this link https://github.com/igorescobar/jQuery-Mask-Plugin/graphs/contributors its better?

File: composer.json (hunk 1 above)
Comment (kamihouse) [inline composer.json]: ok

File: composer.json (hunk 2 above)
Comment (kamihouse) [inline composer.json]: I will leave only the author, I do not know the correct way to point to those responsible.

**Comment by ApeWare** [comment]
Found package on [Packagist](https://packagist.org/packages/igorescobar/jquery-mask-plugin)

Ran
`composer require igorescobar/jquery-mask-plugin`

Getting the following error:
`[Composer\Repository\InvalidRepositoryException]
  No valid composer.json was found in any branch or tag of https://github.com/igorescobar/jQuery-Mask-Plugin.git, could not load a package from it.`

Can you verify this is setup correctly?

**Comment by igorescobar** [comment]
@kamihouse The webhook is active and I would appreciate if you could pass the package over to my account "igorescobar". Thanks for your help and contribution!

**Pull request merged**